bitnami/redis/templates/replicas/hpa.yaml

```
@@ -1,4 +1,4 @@
-{{- if .Values.replica.autoscaling.enabled }}
+{{- if and (eq .Values.architecture "replication") .Values.sentinel.enabled }}
```

Could you change this?
```suggestion
{{- if and (eq .Values.architecture "replication") (not .Values.sentinel.enabled) }}
```